### PR TITLE
move lint for advisory to wolfictl lint

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

**Review by developer-guy — DISMISSED** [review]

**Review by cpanato — CHANGES_REQUESTED** [review]
ok, looks weird

what is the difference between the wolfi-lint and the lint?

if both are the same you need to drop the entire job and not only the line

**Comment by rawlingsj** [comment]
> if both are the same you need to drop the entire job and not only the line

They are slightly different, the `./lint.sh` script was added first but not all checks have been ported over yet.  There was a `wolfictl advisory sync` added since then but rather than using the `./lint.sh` script and compiling wolfictl from source in the GHA, lets use the image instead.

**Review by cpanato — DISMISSED** [review]

File: .github/workflows/wolfictl-lint.yaml [hunk 1]
```diff
@@ -18,4 +18,10 @@ jobs:
...
         entrypoint: wolfictl
-        args: lint
\ No newline at end of file
+        args: lint
+    - name: Advisory sync
+      id: advisory
+      uses: docker://ghcr.io/wolfi-dev/wolfictl:latest
+      with:
+        entrypoint: wolfictl
+        args: "advisory sync-secfixes --warn *.yaml"
```
Comment (luhring) [inline .github/workflows/wolfictl-lint.yaml:27]: @rawlingsj I think we can just drop the file part — the command should default to looking at all files in the current directory

```suggestion
        args: "advisory sync-secfixes --warn"
```

Comment (rawlingsj) [reply .github/workflows/wolfictl-lint.yaml]: YAY it passed and was super fast!

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**